fdbclient/CMakeLists.txt

```diff
@@ -189,6 +189,7 @@ target_link_libraries(fdbclient PUBLIC fdbrpc msgpack)
 add_flow_target(STATIC_LIBRARY NAME fdbclient_sampling SRCS ${FDBCLIENT_SRCS} ADDL_SRCS ${options_srcs})
 add_dependencies(fdbclient_sampling fdboptions fdb_c_options)
 target_link_libraries(fdbclient_sampling PUBLIC fdbrpc_sampling msgpack)
+add_dependencies(fdbclient_sampling_actors fdbclient_actors)
```

Comment:
I don't see a target named `fdbclient_sampling_actors`, did you mean `fdbclient_sampling` (same for `fdbclient_actors` -> `fdbclient`)? Adding a dependency on the non-sampling target for the sampling targets seems ok to me if it will help avoid parallel build problems -- but I haven't seen this issue with clang or gcc, is this an MSBuild problem? Can you specify dependencies for MSBuild itself instead of adding a dependency here?

Reply:
The actors targets are generated by the `add_flow_target` function to prebuild with actorcompiler (from what I understood). You can find them in build.ninja from example.
Indded, the problem occurs only with msbuild, so I can add a specification for the dependency.